.github/workflows/generator_container_slsa3.yml

```diff
@@ -134,3 +160,23 @@ jobs:
             --type slsaprovenance \
             --force \
             "${UNTRUSTED_IMAGE}@${UNTRUSTED_DIGEST}"
+
+      - name: Final outcome
+        id: final
+        env:
+          SUCCESS: ${{ steps.generate-builder.outcome != 'failure' && steps.cosign-install.outcome != 'failure' && steps.login.outcome != 'failure' && steps.sign-prov.outcome != 'failure' }}
+        run: echo "outcome=$([ "$SUCCESS" == "true" ] && echo "success" || echo "failure")" >> $GITHUB_OUTPUT
+
+  # final fails or suceeds based on the value of `inputs.continue-on-error`
+  # and the outcome of previous jobs.
+  final:
+    runs-on: ubuntu-latest
+    needs: [detect-env, generator]
+    # Note: always run even if nedded jobs are skipped.
+    if: always()
+    steps:
+      - name: Final outcome
+        env:
+          SUCCESS: ${{ needs.detect-env.outputs.outcome != 'failure' && steps.generator.outputs.outcome != 'failure' }}
+          CONTINUE: ${{ inputs.continue-on-error }}
+        run: [ "$CONTINUE" == "true" ] || [ "$SUCCESS" == "true" ] || exit 27
```

Comment:
@ianlewis any ida why this is failing but the other similar occurrences (e.g. line 168) are not flagged?

Reply:
Not sure why 168 isn't picked up but I think you'll want to do something like:

```yaml
run: |
  [ "$CONTINUE" == "true" ] || [ "$SUCCESS" == "true" ] || exit 27
```

So it's interpreted as a multi-line string rather than as via YAML's type inference. Probably yamllint thinks you're trying to create a yaml list. e.g.

```yaml
run: ["foo", "bar"]
```

This holds for all the other places where you use `run`.

Reply:
Thanks!